server/index.js

```diff
@@ -44,6 +45,14 @@ app.use(
   })),
 );
 
+// Set express rate limiter
```

Comment:
You have this too late in the application. It needs to occur before line 29. 

The rate limiter is a middleware. So any requests to the backend will pass through it. 
So we want to put this in front of the actual /graphql calls.

Reply:
Oh that makes sense.  Thanks!